<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="mainVerticalLayout">
      <item>
       <widget class="QListWidget" name="questionsListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="answerFormLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="answerLineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="answerLabel">
            <property name="text">
             <string>Answer: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="answerPushButton">
          <property name="text">
           <string>Answer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="currentScoreHorizontalLayout">
        <item>
         <widget class="QLabel" name="currentScoreLabel">
          <property name="text">
           <string>Current score: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentScoreLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="addHorizontalLayout">
        <item>
         <layout class="QFormLayout" name="addFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>Id: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="idLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel">
            <property name="text">
             <string>Text: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="textLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="correctAnswerLabel">
            <property name="text">
             <string>Correct answer: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="correctAnswerLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="scoreLabel">
            <property name="text">
             <string>Score: </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="scoreLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
